README.md

```diff
@@ -1,2 +1,193 @@
 # ast-gh-actions
+
 A library or collection of AST github shared composite actions.
+
+## Development
+
+### Git Pre-commit Hook
+
+The repository uses a pre-commit hook to run linting on all TypeScript projects. To enable:
+
+```bash
+git config core.hooksPath .githooks
```

Comment:
Redundant with the (currently-named) `prepare` command, either should be removed IMO.